src/screens/OtherProfile/index.js

```diff
@@ -669,21 +677,21 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center'
   },
-  textTotal: {
+  textTotal: (isAnon) => ({
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
-    color: colors.bondi_blue,
+    color: isAnon ? COLORS.anon_primary : COLORS.signed_primary,
     paddingRight: 4
-  },
+  }),
   textFollow: {
     fontSize: 14,
-    color: colors.black,
+    color: COLORS.black,
     paddingRight: 4
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values in this section have been updated to use the new `COLORS` object from the `theme` module, which is a good practice for maintaining consistency across the application. However, it's important to ensure that the new colors (`COLORS.anon_primary`, `COLORS.signed_primary`, `COLORS.black`) are defined in the `COLORS` object and they match the intended design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->